protocol/chat.go

```diff
@@ -501,18 +501,18 @@ func CreateCommunityChat(orgID, chatID string, orgChat *protobuf.CommunityChat,
 
 func (c *Chat) DeepLink() string {
 	if c.OneToOne() {
-		return "status-im://p/" + c.ID
+		return "status-app://p/" + c.ID
 	}
 	if c.PrivateGroupChat() {
-		return "status-im://g/args?a2=" + c.ID
+		return "status-app://g/args?a2=" + c.ID
 	}
 
 	if c.CommunityChat() {
-		return "status-im://cc/" + c.ID
+		return "status-app://ccm/" + c.ID
```

Comment:
Not sure if desktop need align this ?

Reply:
Thanks, this impl is incorrect, I'll update it

Reply:
I searched the desktop code, looks like desktop is fine with these changes, no updates needed on our side